### PR TITLE
Update copyright notice

### PR DESCRIPTION
**Pull request by dmdashenkov**
This PR updates the copyright notices in this repository.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol, PTAL.

File: .github/keys/gradle-plugin-portal.secret.properties.gpg [hunk 1]
```diff
@@ -0,0 +1,2 @@
+�	2�.y�:���ҢF���`�J�1$����9������D~�z�e��!yu��GBU��2�K�0�!��	����q=���˂p�������@z���e�qB�>�R#�qj�?�1��m굘U��^�2��*��k��J�	3�w�q84}.
```
Comment (armiol) [inline .github/keys/gradle-plugin-portal.secret.properties.gpg]: I don't think this file is needed here.

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+name: Publish
```
Comment (armiol) [inline .github/workflows/publish.yml]: This workflow is not applicable here. We have another one called `publish-gcloud.yml` instead.

**Review by armiol — APPROVED** [review]
@dmdashenkov please process the comments before merging.

**Pull request merged**